medusa.json

```diff
@@ -0,0 +1,89 @@
+{
+	"fuzzing": {
+		"workers": 10,
+		"workerResetLimit": 50,
+		"timeout": 0,
+		"testLimit": 0,
+		"shrinkLimit": 5000,
+		"callSequenceLength": 100,
+		"corpusDirectory": "",
+		"coverageEnabled": true,
+		"coverageFormats": [
+			"html",
+			"lcov"
+		],
+		"targetContracts": ["FuzzTest"],
+		"predeployedContracts": {},
+		"targetContractsBalances": [],
+		"constructorArgs": {},
+		"deployerAddress": "0x30000",
+		"senderAddresses": [
```

Comment:
I think we are not using these.

Reply:
we are not taking advantage of the various sender addresses in the simple boilerplate example, but they are usually later used in our actor setups for bigger projects, so I lean on leaving them in

what should be the alternative? using only one? I believe medusa wont run if they key is deleted or has an empty array as a value

Reply:
I would use just one only if somehow it enhance the performance.

Reply:
afaik having one vs many doesnt affect performance (as long as having many users doesnt make the domain of possible states grow at a relevant rate, which is not the case for the humble greeter)